### PR TITLE
Add push config

### PR DESCRIPTION
**Pull request by evandeininger**
Add `gitpush` to config, default to use git push unless the user has specified otherwise.

### CODE REVIEW

File: .github/CONTRIBUTING.md [hunk 1]
```diff
@@ -9,7 +9,7 @@ Thanks for considering contributing to the project.
 3. Create a new branch for your changes.
 4. Make your changes and commit them with descriptive commit messages.
 5. Push your changes to your forked repository.
-6. Create a pull request from your branch to the `dev` branch.
+6. Create a pull request from your branch to the `master` branch.
```
Comment (evandeininger) [inline .github/CONTRIBUTING.md:12]:
```suggestion
6. Create a pull request from your branch to the `dev` branch.
```

**Comment by woop** [comment]
Love this.

**Comment by evandeininger** [comment]
@di-sukharev Hello! Is there anything about this I need to update? Should I ditch it? Thanks again for working on this! Huge time saver.

**Comment by evandeininger** [comment]
not enough interest, going to close.

**Comment by kodbilenadam** [comment]
@di-sukharev Can you check this out? 😄

**Comment by zen0bit** [comment]
I am interested...

**Comment by Moret84** [comment]
Interested as well. I am suprised you closed your own PR. Even if the author did not review it for almost two months, he may do it at some point. I think most people might be interested in this feature.

**Comment by jonnyhoff** [comment]
> I am interested...

Me too.

**Comment by Moret84** [comment]
@evandeininger do you mind if I take your branch and submit a new pr ?

**Comment by evandeininger** [comment]
> @evandeininger do you mind if I take your branch and submit a new pr ?
> 
> 
> 
> edit: see #220 

Go ahead! Hope it goes through this time.

**Pull request closed without merging**